### ISSUE TITLE
VAOS Calendar Widget - 508 Findings

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#2671]
This epic will be where we file any 508 findings specific to the calendar widget being developed under PR https://github.com/department-of-veterans-affairs/vets-website/pull/10877

**Comment by lenaecb** [comment]
@1Copenut - same for this epic, should we consolidate all 508 related items for VAOS into a single epic? This one looks like it has some open items so perhaps it needs to stay as is.

**Comment by 1Copenut** [comment]
@lenaecb Let's keep this epic open. I'll roll it into a parent post-launch audit epic, and communicate the over-arching number then. I'm out this afternoon, but will have Slack access if you need to reach me.

**Comment by lenaecb** [comment]
@1Copenut - sounds good!

**Comment by lenaecb** [comment]
Given we've completed the full staging review and all calendar tickets are closed at this point, I am closing this epic. 

Thanks!
